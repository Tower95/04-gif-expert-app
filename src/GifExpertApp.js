import React, { useState } from "react";
import ReactDOM from "react";

import AddCategory from "./components/AddCategory";

const GifExpertApp = ( ) => {

  const [categories, setCategories] = useState(['Dragon Ball', 'Evangelion', 'Full Metal Alchimist']);
  const [input, setinput] = useState("");


  return (<>
    <h2>Gif Expert App</h2>
    <AddCategory setCategories = {setCategories} />
    <hr />
    
    <ol>
      {
        categories.map((category) => {
          return <li key={category}>{category}</li>
        })
      }
    </ol>
  </>

  );
}

export default GifExpertApp;
