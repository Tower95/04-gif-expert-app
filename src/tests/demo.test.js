import React from "react";
import {shallow} from 'enzyme'

import GifExpertApp from'../GifExpertApp'

describe('prubas en <GifExpertApp' , ()=>{
  test('deve de mostrar <GifExpertApp /> correctamente', ()=>{
    const saludo = 'hola Soy Goku'
    const wrapper = shallow( <GifExpertApp/> );
    expect(wrapper).toMatchSnapshot();
  })
})