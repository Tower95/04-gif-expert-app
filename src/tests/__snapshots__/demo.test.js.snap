// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`prubas en <GifExpertApp deve de mostrar <GifExpertApp /> correctamente 1`] = `
<Fragment>
  <h2>
    Gif Expert App
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Dragon Ball"
      key="Dragon Ball"
    />
  </ol>
</Fragment>
`;
